tvb/datatypes/time_series.py

```diff
@@ -37,7 +37,8 @@
 
 from tvb.datatypes import sensors, surfaces, volumes, region_mapping, connectivity
 from tvb.basic.neotraits.api import HasTraits, Attr, NArray, List, Float, narray_summary_info
-
+import numpy
```

Comment:
Do we really do not have numpy already imported here?
I have not tried it, but it surprises me if that is the case

Reply:
Surprisingly, numpy is not imported in this file. This is the reason for this import.

Reply:
Ok. Thanks for checking. Then it is perfectly ok to import it here.